### DISCUSSION TITLE
Solidity Metrics not working on mac

### DISCUSSION THREAD

**Discussion by Teoslaf** [Cyfrin/security-and-auditing-full-course-s23#241] [General]
I tried with cmd + shift + p, for some people worked, but not for me.
Heres the error:
Error loading webview: Error: Could not register service worker: InvalidStateError: Failed to register a ServiceWorker: The document is in an invalid state..

Thank you!

**Comment by EngrPips** [comment]
I always just right-click on the file I want to use `Solidity Metrics` on, and I will select `Solidity Metrics` from the list of options my Mac responds with when I right-click a file. It has always worked for me that way.
